app/src/main/java/com/eveningoutpost/dexdrip/watch/miband/MiBandService.java

```diff
@@ -1096,6 +1260,10 @@ private void resetFirmwareState(Boolean result) {
     }
 
     private void resetFirmwareState(Boolean result, String customText) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            requestConnectionPriority(I.connection, BluetoothGatt.CONNECTION_PRIORITY_BALANCED);
```

Comment:
`I` isn't a understandable name.

Reply:
This name defined in JamBaseBluetoothSequencer class which was not modified by me.